apps/wallet/tailwind.config.js

```diff
@@ -17,16 +17,29 @@ module.exports = {
             colors: {
                 'gradient-blue-start': '#589AEA',
                 'gradient-blue-end': '#4C75A6',
+                'alice-blue': '#F1F8FD',
```

Comment:
If this is a color we're using can the figma be updated to include it? @mystie711